cmd/rm.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	cmdutil "github.com/puppetlabs/wash/cmd/util"
+	"github.com/puppetlabs/wash/plugin"
+)
+
+func rmCommand() *cobra.Command {
+	use, aliases := generateShellAlias("rm")
+	rmCmd := &cobra.Command{
+		Use:     use + " <path> [<path>]",
+		Aliases: aliases,
+		Short:   "Deletes the entries at the specified paths",
+		Long: `Deletes the entries at the specified paths, prompting the user for confirmation
+before deleting each entry.`,
+		Args: cobra.MinimumNArgs(1),
+		RunE: toRunE(rmMain),
+	}
+	rmCmd.Flags().BoolP("recurse", "r", false, "Delete directories (parent entries)")
```

Comment:
Hmm, given the asymmetry between the `fs/delete` endpoint and the `Delete` method's signature (something that isn't an issue in our other actions), I'm starting to think that maybe the `recurse` option isn't such a good idea. Is there a compelling reason for us to enforce it? It seems simple enough to just say "`wrm` removes the given entries" (or, equally, that "rm"'s semantics don't cleanly apply to Wash, which is a good reason to change `wrm` => `delete` so that people don't get confused).

(If we talked about not having `-r` on Wednesday's meeting and I spoke against it, then apologies for that).

Reply:
Updated the issue description, I think it's clear to remove `-r` and rename the command.